### DISCUSSION TITLE
Credit Notes (involving discounts)

### DISCUSSION THREAD

**Discussion by IamBaraa** [lavaloon-eg/ksa_compliance#161] [Q&A]
Hi everyone,

I wanted to ask about the support for ZATCA Phase 2 Credit Notes (those involving discounts after issuing invoices).

For example, a sales invoice was issued for 10,000 and upon payment, the customer requested a discount of 500. May I know how to deal with such scenarios?

Looking forward to your insights!

Thanks!

**Comment by m4mousa** [comment]
we have same issue

**Comment by mhaggag** [comment]
@khloud-Aboelhasan Please discuss with the business and functional teams to see whether we have a working solution.

**Comment by rashedpro** [comment]
I have same issue

**Comment by hamzahabdo** [comment]
Also we have this issue

**Comment by Ahmed-M-Fawzy** [comment]
It is essential to comply with the regulations set by ZATCA. The following actions must be taken:

Preferred Solution: Issue a return invoice for the original invoice, then create a new invoice that includes the discount granted to the customer.
Alternative Solution: Apply the discount at the time of payment, keeping the original invoice unchanged. However, this approach has a drawback: the seller must pay the full tax amount to ZATCA, regardless of the discount given at payment.

**Reply by m4mousa** [reply]
Thanks Ahmed,
practically we grand customer discount against multiple invoices, almost by the end of the year, so we can not make return to all customer invoices then make another invoices

**Comment by jehad2020** [comment]
We need it too , We facing the same situation

**Comment by Ahmed-M-Fawzy** [comment]
@m4mousa @jehad2020 
In such cases, the discount is considered to be applied at the time of payment and is calculated from the sales commissions granted to customers to achieve sales targets. The invoices remain unchanged, and the seller bears the full tax amount on the invoice. 
### **Modifying the invoices could be seen as **fraudulent** by ZATCA**, 
which we do not support, as we adhere to the existing and binding regulations set by ZATCA.

For technical advice and legal opinions, you can contact ZATCA directly. We will certainly comply with any laws that are enacted.
